### PR TITLE
Improve Sentry doc

### LINKED ISSUE #18521: [docs] Using Sentry - Unexpected token: Sentry.Native.*
### Summary

I'm moving from managed to bare workflow and I've run into this error:

`Error:, [SentryError: Native Client is not available, can't start on native.]`

After reading "Expo: Using Sentry" documentation, I've changed the Sentry.init code as documented:

```javascript
import * as Sentry from 'sentry-expo';

Sentry.init({
  dsn: 'YOUR DSN HERE',
  enableInExpoDevelopment: true,
  debug: true, // If `true`, Sentry will try to print out useful debugging information if something goes wrong with sending the event. Set it to `false` in production
});

// Access any @sentry/react-native exports via:
Sentry.Native.*

// Access any @sentry/browser exports via:
Sentry.Browser.*
```

But now I've ran into this error:

```
Android Bundling failed 1699ms
SyntaxError: /home/roberto/Documents/Projetos/caseira/caseira-app/caseira.app/App.js: Unexpected token (18:14)

  16 |
  17 | // Access any @sentry/react-native exports via:
> 18 | Sentry.Native.*
     |               ^
  19 |
  20 | // Access any @sentry/browser exports via:
  21 | Sentry.Browser.*
```

I guess there's something missing on documentation.

### Link to the related docs page

https://docs.expo.dev/guides/using-sentry/

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by BrodaNoel**
# Why
Fix https://github.com/expo/expo/issues/18521

### CODE REVIEW

File: docs/pages/guides/using-sentry.md [hunk 1]
```diff
@@ -65,12 +67,25 @@ Sentry.init({
...
 });
+```
+
+#### Usage
+
+Depending on which platform you are (mobile or web), use the following functions in order to capture/catch exceptions:
 
+```js
 // Access any @sentry/react-native exports via:
 Sentry.Native.*
```
Comment (brentvatne) [inline docs/pages/guides/using-sentry.md:78]:
```suggestion
// Sentry.Native.*
```

File: docs/pages/guides/using-sentry.md [hunk 2]
```diff
@@ -65,12 +67,25 @@ Sentry.init({
...
 
 // Access any @sentry/browser exports via:
 Sentry.Browser.*
```
Comment (brentvatne) [inline docs/pages/guides/using-sentry.md:81]:
```suggestion
// Sentry.Browser.*
```

**Review by brentvatne — APPROVED** [review]

File: docs/pages/guides/using-sentry.md [hunk 3]
```diff
@@ -55,6 +55,8 @@ cmd={['$ expo install expo-application expo-constants expo-device expo-updates @
 
 ### Step 2: Code
 
+#### Config
```
Comment (brentvatne) [inline docs/pages/guides/using-sentry.md:58]:
```suggestion
#### Configuration
```

**Comment by BrodaNoel** [comment]
@brentvatne done

File: docs/pages/guides/using-sentry.md [hunk 4]
```diff
@@ -65,12 +67,25 @@ Sentry.init({
   enableInExpoDevelopment: true,
   debug: true, // If `true`, Sentry will try to print out useful debugging information if something goes wrong with sending the event. Set it to `false` in production
 });
+```
+
+#### Usage
+
+Depending on which platform you are (mobile or web), use the following functions in order to capture/catch exceptions:
```
Comment (kbrandwijk) [inline docs/pages/guides/using-sentry.md:74]:
```suggestion
Depending on which platform you are on (mobile or web), use the following functions in order to capture/catch exceptions:
```

Comment (kbrandwijk) [reply docs/pages/guides/using-sentry.md:74]: Aside from that, you are not listing functions, so 'the following functions' doesn't really apply. Also, those functions are not _just_ for capturing/catching exceptions, there's methods in there for instrumentation, performance, etc. so a more generic description would be better here.

**Review by kbrandwijk — CHANGES_REQUESTED** [review]

File: docs/pages/guides/using-sentry.md [hunk 5]
```diff
@@ -55,6 +55,8 @@ cmd={['$ expo install expo-application expo-constants expo-device expo-updates @
 
 ### Step 2: Code
 
+#### Configuration
+
 Add the following to your app's main file (usually `App.js`):
```
Comment (amandeepmittal) [inline docs/pages/guides/using-sentry.md:60]:
```suggestion
Add the following to your app's main file such as **App.js**:
```

Unrelated to your PR, here is a small change to [bold the filenames as per Expo's writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md).

File: docs/pages/guides/using-sentry.md [hunk 6]
```diff
@@ -65,12 +67,25 @@ Sentry.init({
   enableInExpoDevelopment: true,
   debug: true, // If `true`, Sentry will try to print out useful debugging information if something goes wrong with sending the event. Set it to `false` in production
 });
+```
+
+#### Usage
+
+Depending on which platform you are on (mobile or web), use the following functions in order to capture/catch exceptions:
```
Comment (amandeepmittal) [inline docs/pages/guides/using-sentry.md:74]:
```suggestion
Depending on which platform you are on (mobile or web), use the following functions to access any `@sentry/*` exports for instrumentation, performance, and so on:

- For React Native, access any `@sentry/react-native` exports with `Sentry.Native.*`
- For web, access any `@sentry/browser` exports with `Sentry.Browser.*`
```

As per Kim's suggestion, let's be more verbose about accusing any exports from `@sentry` packages.

File: docs/pages/guides/using-sentry.md [hunk 7]
```diff
@@ -65,12 +67,25 @@ Sentry.init({
...
-Sentry.Browser.*
+// Sentry.Browser.*
+
+// Example:
+try {
+  // your code
+} catch (error) {
+  Sentry.Native.captureException(error);
+}
 ```
```
Comment (amandeepmittal) [inline docs/pages/guides/using-sentry.md:89]: The example then, could be written as:

The following example uses `captureException()` from `Sentry.Native.*` to capture errors:

```js
try {
  // your code
} catch (error) {
  Sentry.Native.captureException(error);
}
```

File: docs/pages/guides/using-sentry.md [hunk 8]
```diff
@@ -65,12 +67,25 @@ Sentry.init({
...
 
 ### Step 3: App Config
```
Comment (amandeepmittal) [inline docs/pages/guides/using-sentry.md:91]:
```suggestion
### Step 3: App Configuration
```

Let's also change `Config` here from `Configuration` for consistent writing.

**Review by amandeepmittal — CHANGES_REQUESTED** [review]
Here are some suggestions based on your changes to improve the doc and follow some of our Writing Style guide for consistency.

**Comment by BrodaNoel** [comment]
Changes applied.
Anything else I could fix?
I'm not an english native speaker, so, if there is some working that "should be improved", please provide me the exact correct phrase because what I wrote was my best attempt :P

**Review by amandeepmittal — APPROVED** [review]
Everything looks great! Thanks for improving this doc @BrodaNoel 👍

**Review by kbrandwijk — APPROVED** [review]

**Pull request merged**